.github/labeler.yml

```diff
@@ -15,3 +15,7 @@ api:
 github:
```

Comment:
I'd say the `github` label is unclear tbh. We should just use the `ci` label here imo or, if this is a different _category_ of labelling to what we're doing with conv commits [here](https://github.com/Flagsmith/flagsmith/pull/4084/files#diff-88b40629df399d3611837cb436b6d43f743791f1f5944ea002bb76bf3713046aR32), maybe we should use `workflows` instead?